src/senaite/core/browser/controlpanel/setupview.py

```diff
@@ -81,6 +81,10 @@ def setupitems(self):
         # filter out items
         items = filter(lambda item: not item.exclude_from_nav, items)
 
+        if self.setup.getEnableGlobalAuditlog() is False:
```

Comment:
PEP8's [*don’t compare boolean values to True or False using ..*](https://peps.python.org/pep-0008/#programming-recommendations). Better use: 

```python
if not self.setup.getEnableGlobalAuditlog()
```

Reply:
Uh, it is even declared as "Worse". Thanks for pointing it out!

Reply:
partially off-topic: would it make sense to add a flake8 job to the Travis CI setup to catch these?

Reply:
> partially off-topic: would it make sense to add a flake8 job to the Travis CI setup to catch these?

We did that once and were flooded by PEP8 messages, so we sticked to the local flake8 checker of the editors. Wondering now, why my flake8 did not complain about it...

Reply:
> Wondering now, why my flake8 did not complain about it...

flake8 does not consider this a problem:

> E712 comparison to False should be 'if cond is False:' or 'if not cond:'

(see also https://www.flake8rules.com/rules/E712.html)

Reply:
I have followed up on this in https://github.com/senaite/senaite.core/pull/2077/. Please let me know what you think about this approach.